core/docs/d7yxc-operator_manual.md

```diff
@@ -102,6 +102,22 @@ To ensure that an SDK will be used with a compatible CLI and runner:
 1. If using a pre-installed CLI, install the CLI at that version as detailed in [Versioning](#versioning).
 1. Once the runner and/or CLI are setup, you are safe to upgrade your SDK to the newest version.
 
+### Can I run the Dagger Engine as a "rootless" container?
+
+No. "Rootless mode" means running the Dagger Engine as a container without the `--privileged` flag. In this case, the container would not have "root" privileges for all devices on the host system. Currently, the Dagger Engine cannot be run as a rootless container; network, platform and volume constraints related to rootless usage would significantly limit its capabilities and performance.
+
+#### Platform constraints
+
+The Dagger Engine uses Buildkit. In order to run it in rootless mode, it is necessary to use [Rootlesskit](https://github.com/rootless-containers/rootlesskit), an experimental tool that aims to replace the "root" user with a "fake root" user using user namespaces. However, depending on the host operating system, some features might not be fully supported.Since Dagger is intended to run consistently everywhere, introducing platform-specific incompatibilities would be contrary to this goal.
+
+#### Volume constraints
+
+The Dagger Engine relies on the `overlayfs` snapshotter for efficient operation of mounted filesystems. In order to run it in rootless mode, it is necessary to use [`fuse-overlayfs`](https://github.com/containers/fuse-overlayfs), a rootless implementation of `overlayfs`. However, optimal configuration of `fuse-overlayfs` depends on the kernel capabilities, its own configuration, `fuse` availability and upstream kernel patches specific to the host distribution. In the absence of all these, correctly configured, you will experience noticeable slowdown, either with `fuse-overlayfs` or the default snapshotter. Since Dagger aims to accelerate your CI/CD pipeline, reducing performance also reduces the value provided by Dagger and leads to unstable behaviour.
+
+#### Network constraints
+
+Running the Dagger Engine in rootless mode constrains network management. It is possible to use [slirp](https://github.com/rootless-containers/slirp4netns) as a workaround, but this tool significantly decreases network performance in addition to being extremely complex to setup. This [comparison table of network drivers](https://github.com/rootless-containers/rootlesskit/blob/master/docs/network.md#network-drivers) shows that `slirp` is at least five times slower than a root-privileged network driver.
```

Comment:
```suggestion
Running the Dagger Engine in rootless mode constrains network management due to the fact that it's not possible for a rootless container to move a network device from the host network namespace to its own network namespace. 

It is possible to use userspace TCP/IP implementations such as [slirp](https://github.com/rootless-containers/slirp4netns) as a workaround, but they often significantly decrease network performance. This [comparison table of network drivers](https://github.com/rootless-containers/rootlesskit/blob/master/docs/network.md#network-drivers) shows that `slirp` is at least five times slower than a root-privileged network driver.

Newer options for more performant userspace network stacks have arisen in recent years, but they are generally either reliant on relatively recent kernel versions or in a nascent stage that would require significant validation around robustness+security.
```

Added a little more detail on the specific constraint we hit. Also clarified that `slirp` is only one option.